### DISCUSSION TITLE
Getting "unknown command" for module command

### DISCUSSION THREAD

**Discussion by ccollie** [valkey-io/valkey#1331] [General]
In my module, I'm getting the error above for a command event though it appears when I do a `command list`. Any ideas on why this happens ?

**Comment by madolson** [comment]
What command are you trying to run?

**Reply by ccollie** [reply]
I've since resolved the issue. This can be marked as closed.

**Comment by hwware** [comment]
Because the problem it looks like solved, I close it.
